CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 4.25.1
 
+* BraintreeCore
+  * Provide more error information on Browser Switch failures
```

Comment:
nit: is it worth calling out the flows this will update specifically? Something like "Browser Switch failures in PayPal, Native Checkout, and Three D Secure flows"

Reply:
Definitely yeah I'll update.